lib/OpenCloud/LoadBalancer/Service.php

```diff
@@ -51,11 +52,20 @@ public function loadBalancer($id = null)
      */
     public function loadBalancerList($detail = true, array $filter = array())
     {
+        $options = $this->makeResourceIteratorOptions($this->resolveResourceClass('LoadBalancer'));
+
+        if (isset($filter['limit'])) {
+            $options['limit.page'] = $filter['limit'];
```

Comment:
I guess they aren't needed. I was thinking of [these lines](https://github.com/rackspace/php-opencloud/blob/working/lib/OpenCloud/Common/Collection/PaginatedIterator.php#L275-L277) and wanted to have the iterator set the query value rather than couple it to the user's input. But thinking about it, it doesn't matter: either way the query is set and the URL will have the appropriate value.